Sources/TuistSupport/CoreDataVersionExtractor.swift

```diff
@@ -14,11 +14,11 @@ public final class CoreDataVersionExtractor {
             let dict = try decoder.decode([String: String].self, from: data)
             guard
                 let currentVersionName = dict["_XCCurrentVersionName"],
-                let versionNumberString = currentVersionName.split(separator: ".").first
+                currentVersionName.hasSuffix(".xcdatamodel")
             else {
                 throw CoreDataVersionExtractorError.couldNotExtractVersionFrom(filePath)
             }
-            return String(versionNumberString)
+            return String(currentVersionName.dropSuffix(".xcdatamodel"))
```

Comment:
Is wrapping `currentVersionName.dropSuffix(".xcdatamodel")` in a String initializer necessary? 
`dropSuffix` returns a string, am I right?

Reply:
Woopsie ;-)